src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -475,26 +473,21 @@ private async Task<bool> NewerManifestPackageExists(ManifestId manifest)
 
         public IEnumerable<ManifestVersionUpdate> ParseRollbackDefinitionFiles(IEnumerable<string> rollbackFilePaths)
         {
+            var zeroVersion = new ManifestVersion("0.0.0");
```

Comment:
It would be good to add a comment explaining that the previous version isn't actually used anymore so we just set it to 0 to avoid having to resolve the actual manifest versions (which can fail with global.json scenarios).

Reply:
I was mulling this over last night and realized that we do actually use this in the MSI-based installer in two ways: to plan the package (upgrade, downgrade, or 'none'?) and to roll back if we hit an exception (manifestUpdate.Reverse()...)

I'm not sure what the ideal answer is for that scenario. We'd need some way to figure out what's currently installed without erroring...

/cc: @joeloff

Reply:
I think we can fix the MSI installer code.  We don't upgrade or downgrade from an MSI perspective anymore, with side-by-side manifests the MSIs for different versions of the same manifest are not MSI upgrades, they're not related.

Reply:
I'm not sure what your suggestion is?